integrations/api_branch_test.go

```diff
@@ -115,24 +115,74 @@ func testAPICreateBranch(t *testing.T, oldBranchName string, branchName string,
 }
 
 func TestAPICreateBranch(t *testing.T) {
-	defer prepareTestEnv(t)()
-
-	// Create new_branch from default branch
-	testAPICreateBranch(t, "", "new_branch", http.StatusCreated)
+	onGiteaRun(t, testAPICreateBranches)
+}
 
-	// Create new_branch from a given branch
-	testAPICreateBranch(t, "master", "new_branch_2", http.StatusCreated)
+func testAPICreateBranches(t *testing.T, giteaURL *url.URL) {
+	defer resetFixtures(t)
```

Comment:
Nope that needs to go back to the start of the loop.

Reply:
We actually want to keep the environment going to test to create a branch when it already exists don't we ?